src/Divider.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+import { space, width, borderColor, propTypes } from 'styled-system'
+
+const Divider = styled.hr`
+  ${space}
+  ${width}
+  border: 0;
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
+  ${borderColor}
```

Comment:
Super trivial, but the template string(s) can be grouped together at top while the border styles with hard-coded styles at the bottom? (Just a tiny bit organization)

Reply:
I think this is fine as-is – `border-color` needs to be declared at the end to override the `border` shorthand above. I think generally with styled components, functions at the end of the block makes more sense. This also has implications with media queries